search/include/pcl/search/search.h

```diff
@@ -413,7 +411,7 @@ namespace pcl
           }
           
           bool 
-          operator () (int first, int second) const
+          operator () (index_t first, index_t second) const
```

Comment:
Are these indices?

Reply:
They are used to access a vector of floats two lines further down.